src/components/views/read/cvForm/Education.tsx

```diff
@@ -36,14 +38,27 @@ const Education: FC<Props> = ({
   educationData,
   onCvValuesChange,
 }) => {
+  const { postAppData, patchAppData, deleteAppData, appDataArray } =
+    useAppDataContext();
+  const educationInfoObject = appDataArray.find(
+    (obj) => obj.type === 'educationInfo',
+  );
+  const handlePost = (newdata: EducationInfoObj): void => {
+    postAppData({ data: newdata, type: 'educationInfo' });
+  };
+  const handlePatch = (dataObj: any, newData: EducationInfoObj): void => {
```

Comment:
Do not use any, I am guessing you would receive the appData here, which has type: `AppData`.